web/pf4/src/components/ChartWithLegend.tsx

```diff
@@ -171,6 +177,10 @@ class ChartWithLegend extends React.Component<Props, State> {
             height={legend.height}
             width={this.state.width}
             itemsPerRow={legend.itemsPerRow}
+            style={{
+              data: { cursor: 'pointer' },
+              labels: { cursor: 'pointer' }
```

Comment:
Oh, nice !

Reply:
yes that's a good point for victory charts, it exposes many stuff that we can tweak